libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/map/NavigationMapboxMap.java

```diff
@@ -286,6 +316,7 @@ public NavigationCamera retrieveCamera() {
    */
   public void updateCameraTrackingMode(@NavigationCamera.TrackingMode int trackingMode) {
     mapCamera.updateCameraTrackingMode(trackingMode);
+    mapFpsDelegate.updateCameraTracking(trackingMode);
```

Comment:
It seems like the SDK could make use of something along the lines of `NavigationCamera#StateListener` which would broadcast camera mode updates. Not only it could be exposed, but also implemented internally by the likes of `MapFpsDelegate`.

Reply:
@LukasPaczos good idea, I added this with the latest code.